src/generated/java/com/prowidesoftware/swift/model/field/SRU2024FieldEnum.java

```diff
@@ -0,0 +1,746 @@
+    FIELD_102("102"),
+    FIELD_103("103"),
+    FIELD_104("104"),
+    FIELD_105("105"),
+    FIELD_106("106"),
+    FIELD_107("107"),
+    FIELD_108("108"),
+    FIELD_109("109"),
+    FIELD_110("110"),
+    FIELD_111("111"),
+    FIELD_113("113"),
+    FIELD_114("114"),
+    FIELD_115("115"),
+    FIELD_116("116"),
+    FIELD_117("117"),
+    FIELD_118("118"),
+    FIELD_119("119"),
+    FIELD_120("120"),
+    FIELD_121("121"),
+    FIELD_122("122"),
+    FIELD_123("123"),
+    FIELD_124("124"),
+    FIELD_125("125"),
+    FIELD_126("126"),
+    FIELD_127("127"),
+    FIELD_128("128"),
+    FIELD_129("129"),
+    FIELD_130("130"),
+    FIELD_132("132"),
+    FIELD_133("133"),
+    FIELD_134("134"),
+    FIELD_135("135"),
+    FIELD_136("136"),
+    FIELD_137("137"),
+    FIELD_140("140"),
+    FIELD_141("141"),
+    FIELD_142("142"),
+    FIELD_143("143"),
+    FIELD_144("144"),
+    FIELD_151("151"),
+    FIELD_152("152"),
+    FIELD_153("153"),
+    FIELD_165("165"),
+    FIELD_171("171"),
+    FIELD_172("172"),
+    FIELD_173("173"),
+    FIELD_174("174"),
+    FIELD_175("175"),
+    FIELD_177("177"),
+    FIELD_179("179"),
+    FIELD_199("199"),
+    FIELD_202("202"),
+    FIELD_203("203"),
+    FIELD_204("204"),
+    FIELD_206("206"),
+    FIELD_208("208"),
+    FIELD_209("209"),
+    FIELD_242("242"),
+    FIELD_243("243"),
+    FIELD_251("251"),
+    FIELD_252("252"),
+    FIELD_253("253"),
+    FIELD_254("254"),
+    FIELD_255("255"),
+    FIELD_256("256"),
+    FIELD_257("257"),
+    FIELD_258("258"),
+    FIELD_259("259"),
+    FIELD_260("260"),
+    FIELD_263("263"),
+    FIELD_264("264"),
+    FIELD_270("270"),
+    FIELD_271("271"),
+    FIELD_272("272"),
+    FIELD_280("280"),
+    FIELD_281("281"),
+    FIELD_301("301"),
+    FIELD_302("302"),
+    FIELD_303("303"),
+    FIELD_304("304"),
+    FIELD_305("305"),
+    FIELD_307("307"),
+    FIELD_311("311"),
+    FIELD_312("312"),
+    FIELD_313("313"),
+    FIELD_329("329"),
+    FIELD_330("330"),
+    FIELD_331("331"),
+    FIELD_332("332"),
+    FIELD_333("333"),
+    FIELD_334("334"),
+    FIELD_335("335"),
+    FIELD_336("336"),
+    FIELD_337("337"),
+    FIELD_338("338"),
+    FIELD_339("339"),
+    FIELD_340("340"),
+    FIELD_341("341"),
+    FIELD_342("342"),
+    FIELD_343("343"),
+    FIELD_344("344"),
+    FIELD_345("345"),
+    FIELD_346("346"),
+    FIELD_347("347"),
+    FIELD_348("348"),
+    FIELD_349("349"),
+    FIELD_401("401"),
+    FIELD_405("405"),
+    FIELD_421("421"),
+    FIELD_422("422"),
+    FIELD_423("423"),
+    FIELD_424("424"),
+    FIELD_425("425"),
+    FIELD_431("431"),
+    FIELD_432("432"),
+    FIELD_433("433"),
+    FIELD_434("434"),
+    FIELD_443("443"),
+    FIELD_451("451"),
+    FIELD_461("461"),
+    FIELD_503("503"),
+    FIELD_533("533"),
+    FIELD_619("619"),
+    FIELD_999("999");
+
+    private final String code;
+
+    SRU2024FieldEnum(String code) {
+        this.code = code;
+    }
+
+    public String getCode() {
```

Comment:
rename to "fieldName" and implement it as a simple name().substring... using the actual enum value. You do not need the private constructor.

Reply:
Ok